web/src/components/base/ToggleButton.tsx

```diff
@@ -11,13 +11,18 @@ const StyledButton = styled.button<{ $active: boolean | null }>`
     0 1px 4px 1px rgba(0, 0, 0, 0.15);
   border-radius: 18px;
   width: ${toggleButtonWidth}px;
-  height: 80px;
+  height: 100px;
   background-color: ${props => (props.$active ? props.theme.colors.themeColor : props.theme.colors.backgroundColor)};
   color: ${props => props.theme.colors.textSecondaryColor};
   padding: 8px;
   cursor: pointer;
 `
 
+const Text = styled(StyledSmallViewTip)`
+  white-space: normal;
+  word-break: break-word;
```

Comment:
Why have you put this here and not inside StyledSmallViewTip? Wouldn't other usages of `StyledSmallViewTip` have the same problem?

Reply:
good find. changed this in last commit